File: src/client/src/components/playoffPool/models/PlayoffPool.ts
export interface PlayoffPool {
  teams: PlayoffPoolTeam[];
  players: PlayoffPoolPlayer[];
}

export interface PlayoffPoolTeam {
  franchiseId: string;
  name: string;
  rank: number;
  points: number;
  gamesPlayed: number;
  playersRemaining: number;
}

export interface PlayoffPoolPlayer {
  playerId: number;
  name: string;
  franchiseId: string;
  games: PlayoffPoolPlayerGame[];
  points: number;
  gamesPlayed: number;
  eliminated: boolean;
  pick: number;
}

export interface PlayoffPoolPlayerGame {
  points: number;
  passYards: number;
  passTds: number;
  rushYards: number;
  rushTds: number;
  recYards: number;
  recTds: number;
  Interceptions: number;
  FumblesLost: number;
}

File: src/server/GibsonsLeague.Core/Models/PlayoffPool.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace GibsonsLeague.Core.Models
{
    public class PlayoffPool
    {
        public ICollection<PlayoffPoolTeam> Teams { get; set; } = new List<PlayoffPoolTeam>();
        public ICollection<PlayoffPoolPlayer> Players { get; set; } = new List<PlayoffPoolPlayer>();
    }
    public class PlayoffPoolTeam
    {
        public Guid FranchiseId { get; set; }
        public string Name { get; set; }
        public double Points { get; set; } = 0;
        public int GamesPlayed { get; set; } = 0;
        public int PlayersRemaining { get; set; } = 0;
    }
    public class PlayoffPoolPlayer
    {
        [Key]
        public int PlayerId { get; set; }
        public string Name { get; set; }
        public Guid FranchiseId { get; set; }
        public double Points { get; set; } = 0;
        public int GamesPlayed { get; set; } = 0;
        public bool Eliminated { get; set; } = false;
        public int Pick { get; set; } = 0;
    }
}
